programming/opencv-installation.md

```diff
@@ -0,0 +1,33 @@
+
+# Opencv Installation Process 
+Here are the steps, that you will **run in a terminal shell** that will get you ready to run the opencv library and vision track!
+## For Linux 
+****************************** 
+Make sure your version of python is up to date 
+```
+sudo apt-get install python3-pip
```

Comment:
Do you want to include the instructions for other distributions, or only Ubuntu/Debian for now?